openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc

```diff
@@ -3397,7 +3397,8 @@ SOP_OpenVDB_Rasterize_Points::cookVDBSop(OP_Context& context)
             // Setup VEX context
 
 #if UT_MAJOR_VERSION_INT >= 17
-            OP_Caller caller(this, DEP_ContextOptionsReadHandle{});
+            OP_Caller caller(this, context.getContextOptionsStack(),
+                     context.getContextOptions());
```

Comment:
Looks like this was updated once, unsure if this was for a previous or newer version than 17.0.336?

Reply:
It was for an earlier version, probably 17.0.140.

Reply:
New code matches the current version in Houdini; apologies for not having sent it back as a PR ourselves.